cmd/diff.go

```diff
@@ -116,6 +130,20 @@ func diffImages(image1Arg, image2Arg string, diffArgs []string) error {
 	return nil
 }
 
+func diffFile(imageMap map[string]*pkgutil.Image, image1Arg, image2Arg string) error {
+
+	image1FilePath := imageMap[image1Arg].FSPath + filename
+	image2FilePath := imageMap[image2Arg].FSPath + filename
+
+	diff, err := util.DiffFile(image1FilePath, image2FilePath, image1Arg, image2Arg)
+	if err != nil {
+		return err
+	}
+	diff.Filename = filename
```

Comment:
Can you pass filename into `util.DiffFile` so you don't have to set this after the fact?